tilequeue/cache/redis_cache_index.py

```diff
@@ -18,7 +18,7 @@ def intersect(self, coords, tiles_of_interest=None):
                 yield coord
 
     def remove_tiles_of_interest(self, coord_ints):
-        return self.redis_client.srem(self.cache_set_key, coord_ints)
+        return self.redis_client.srem(self.cache_set_key, *coord_ints)
```

Comment:
It's currently [getting called in groups of 1000](https://github.com/tilezen/tilequeue/blob/c8160d77b4ab859d10e1e8854319238bfe2f784b/tilequeue/command.py#L1073-L1074). It's probably a good thing to document, though.